form-libraries/formik.mdx

```diff
@@ -0,0 +1,84 @@
+---
+name: Formik
+menu: 4. Form Libraries
+route: /formik
+---
+
+# Integrating Fonk with React Formik
+
+[Formik](https://github.com/jaredpalmer/formik) is a great library to manage forms state.
+
+Adding **Fonk** to **Formik** allows you to seamlessly add validation capabilities to **Formik**.
+
+In order to use **Fonk** with **Formik** you will have to install **fonk-formik** adaptor:
+
+```bash
+npm install @lemoncode/fonk @lemoncode/fonk-formik--save
```

Comment:
Add white space between `@lemoncode/fonk-formik` and `--save`

Reply:
done